### DISCUSSION TITLE
Tidying up data from Query

### DISCUSSION THREAD

**Discussion by PaulMcF1987** [Budibase/budibase#6186] [Help]
I have run a Query to gather particular data.

`SELECT WLE, count(WLE) FROM TradeJournal GROUP by WLE`

This gives the information that I am after. I then, via the data provider filtered the results to show only `Win`, `Lose`, or `Even`

The problem I have is that, when displayed in a `Paragraph` Element, it displays as `[{"WLE":"Win","count(WLE)":1}]` if the full expression, All I want to display is the number.

In other software, I have been able to display only the number by calling the query using something like 
`{{
  WLE.data.map((row) => {
      return { value: WLE.data };
  });
}}`

I have tried this but no luck.

Can anyone advise of they have managed to achieve the desired outcome?

-------------------------------------------------------------------------------------------------------------------------

I think I may have answered the above question below.

What I cannot figure out though is how to change the color of the text dependent on the figure shown.

I have tried using the conditions...

`Update Setting > Color  'Green' > if x is more than Number 0`
`Update Setting > Color  'Red' > if x is less than Number 0`
`Update Setting > Color  'Blue' > if x equals Number 0`

The number displayed is 41 but it is displayed in blue.

Could it be something to do with the way I am running the query? Is it not being recognised as a number?

**Comment by PaulMcF1987** [comment]
I think I managed to solve it using `{{ pluck WLE.Rows "count(WLE)" }}`
If there is a better way, could someone please share?

**Reply by shogunpurple** [reply]
That way works - you could also use JavaScript to do it:
`return $("WLE.Rows")[0]["count(WLE)"]`

**Comment by PaulMcF1987** [comment]
@shogunpurple 

Thank you for your answer. The issue I am having is changing the color of the text based on the number 

Positive - green, negative - red, Zero - Blue

I tried the javascript and handlebars but regardless of the number it colours it Blue.
I am assuming that this means that the number is not being recognised as a number.
